submissions/mustbefail/oop/index.js

```diff
@@ -0,0 +1,88 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, hands, legs, friends, phrase) {
+    this.friends = friends;
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.hands = hands;
+    this.legs = legs;
+    this.phrase = phrase;
+  }
+  saying() {
+    return this.phrase;
+  }
+}
+
+class Cat extends Inhabitant {
+  constructor(name, gender, friends, phrase) {
+    super('cat', name, gender, 0, 4, friends, phrase);
+  }
+}
+
+class Dog extends Inhabitant {
+  constructor(name, gender, friends, phrase) {
+    super('dog', name, gender, 0, 4, friends, phrase);
```

Comment:
This says 'create a dog with 0 hands and etc' but dogs do not have any hands by design.